packages/w/wlogout/pspec_x86_64.xml

```diff
@@ -0,0 +1,53 @@
+<PISI>
+    <Source>
+        <Name>wlogout</Name>
+        <Homepage>https://github.com/ArtsyMacaw/wlogout</Homepage>
+        <Packager>
+            <Name>Facundo Ciruzzi</Name>
+            <Email>ciruzzifacundo@gmail.com</Email>
+        </Packager>
+        <License>MIT</License>
+        <PartOf>desktop</PartOf>
+        <Summary xml:lang="en">A wayland based logout menu</Summary>
+        <Description xml:lang="en">wlogout is a logout menu for wayland environments
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>wlogout</Name>
+        <Summary xml:lang="en">A wayland based logout menu</Summary>
+        <Description xml:lang="en">wlogout is a logout menu for wayland environments
+</Description>
+        <PartOf>desktop</PartOf>
+        <Files>
+            <Path fileType="config">/etc/wlogout/layout</Path>
+            <Path fileType="config">/etc/wlogout/style.css</Path>
```

Comment:
What does this file do? Can we move it to `/usr` ?

Reply:
Just default config, move them into `/usr` means patch the code, could be it deleted and expect than users create them into their `/home`? or delete them and move them into `/.config` by default?

Reply:
From staff discussion, this packages needs to be patched to support our stateless model. Default config files should be shipped in `/usr/share/defaults`. This is also hard requirement when we move to Serpent, as we have to deal with other 157 offending packages that also does this.

Reply:
okay, i will do it